### DISCUSSION TITLE
unable to connect gnb and ue  error(Tx while waiting for EOB, timed out...)

### DISCUSSION THREAD

**Discussion by rajugbv** [srsran/srsRAN_Project#141] [General Help] [answered]
Hi everyone,

I am trying to implement an SA (Standalone) project for GNB (gNodeB). I am using USRP B210 for the GNB and USRP B200 for the UE (User Equipment).

I have installed the required packages for Open5GS and built the code for SRSUE, GNB, and the core network. However, despite working on establishing the network for the last 2 months, I have been unable to do so successfully. Could you please review the following information and let me know if you notice any issues?

Thank you in advance.

![image](https://github.com/srsran/srsRAN_Project/assets/41448193/33ca10f5-e696-41e0-90b2-f7be336fb3e7)

![WhatsApp Image 2023-07-24 at 1 28 27 PM](https://github.com/srsran/srsRAN_Project/assets/41448193/00af994a-a6e0-4454-9011-9df7d4517644)
![WhatsApp Image 2023-07-24 at 1 10 09 PM](https://github.com/srsran/srsRAN_Project/assets/41448193/0b5dac96-cea4-4d79-a579-2dae0d4a03ba)

[ue_debug_files.zip](https://github.com/srsran/srsRAN_Project/files/12148565/ue_debug_files.zip)

[gnb_debug_files.zip](https://github.com/srsran/srsRAN_Project/files/12148462/gnb_debug_files.zip)

**Comment by pgawlowicz** [comment, marked as answer]
Please use the newest releases of the srsRAN_Project (gNB) and srsRAN_4G (srsUE), and follow the updated [tutorial](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html)
